csharp/src/Apache.Arrow.Flight.Sql/CancelFlightInfoResult.cs

```diff
@@ -0,0 +1,96 @@
+using Google.Protobuf;
+using Google.Protobuf.Reflection;
+
+namespace Apache.Arrow.Flight.Sql;
+
+public enum CancelStatus
+{
+    Unspecified = 0,
+    Cancelled = 1,
+    Cancelling = 2,
+    NotCancellable = 3,
+    Unrecognized = -1
+}
+
+public sealed class CancelFlightInfoResult : IMessage<CancelFlightInfoResult>
```

Comment:
We shouldn't shadow the proto buf class like this. If we need to extend it we should use a partial class to add what is needed

Reply:
There's no implementation of `CancelFlightInfoResult ` class thus I need to understand what we have to do with this?
@jeremyosterhoudt

Reply:
My concern is we are shadowing the `CancelFlightInfoResult` proto by doing this.  We should just change the class to a partial class and remove the extension of `IMessage<CancelFlightInfoResult>`